extensions/panache/hibernate-orm-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/orm/rest/data/panache/deployment/entity/Collection.java

```diff
@@ -18,6 +18,8 @@
 
     public String name;
 
+    public Long age;
```

Comment:
@gytis I added it for testing purposes to show that PUT will update a whole entity and PATCH just what we want, same for repository

Reply:
I understand and I think it's good that you've added it. I'm just saying that having this field as a primitive (`long` instead of `Long`), would also allow us to verify that primitive values are updated correctly.

Reply:
@gytis oh, sorry, my bad, got it